mcc/odin/odin/snapshot_list_test.go

```diff
@@ -17,3 +21,39 @@ var exampleSnapshot1 = &rds.DBSnapshot{
 	MasterUsername:       aws.String("owner"),
 	Status:               aws.String("available"),
 }
+
+type listSnapshotsCase struct {
+	testCase
+	name       string
+	snapshots  []*rds.DBSnapshot
+	instanceID string
+}
+
+var listSnapshotsCases = []listSnapshotsCase{
+	// No snapshots for any instance
+	{
+		testCase: testCase{
+			expected:      []*rds.DBSnapshot{},
+			expectedError: "",
+		},
+		name:       "No snapshots for any instance",
+		snapshots:  []*rds.DBSnapshot{},
+		instanceID: "",
+	},
+}
+
+func TestListSnapshots(t *testing.T) {
```

Comment:
`*testing.T` is the interface to Go's testing library.
It is mostly used to access methods to report errors, failures, and so on.
There's no need to be verbose on this, since it's always like this.